### DISCUSSION TITLE
Homebrew in Linux

### DISCUSSION THREAD

**Discussion by jrock2004** [nicknisi/dotfiles#186] [General]
I am seriously thinking that maybe home-brew in Linux is just not worth it. Its kinda upsetting because I think one place to have my stuff to install, but its just too slow. Anyone else thinking this or is it just me

**Comment by yarthur** [comment]
How are you using it? Are you trying to get a Brewfile set up, are you trying to run the update commands, or just when installing any old package?

And how slow is too slow?

**Reply by jrock2004** [reply]
When I run the install script to setup a new machine and I go to the part where its install items. There are some apps that takes 5 minutes before it moves to the next item

**Reply by yarthur** [reply]
I guess I’m not setting up new machines that often. 🤷‍♂️ I, myself, expect that initial setup to take a while, and often a couple tries (I clearly have some kinks to work out, but not a lot of testing opportunities).

I like that Brew gives a clean, easy way to install a given set of packages. I’m not handy with shell scripts, so it works just fine with me. I assume you’d use apt or similar to install most things, and if that works better for you, go with it.

**Reply by yarthur** [reply]
(I should say, 5 minutes would absolutely be too long if I ran it regularly. Like, multiple times a day regularly, but I’ve also been overly-tolerant of some janky systems.)
